README.md

```diff
@@ -420,7 +429,8 @@ func main() {
 	plugins := []transportd.NewComponent{
 		components.Metrics,
 		components.AccessLog,
-		components.ASAPValidate,
+    components.ASAPValidate,
```

Comment:
Nit: can you put make these 2 lines consistent with the others in terms of indentation